bcgov/bcrs-shared-components
# updated storybook and lerna + created DetailComponent component

Issue #: /bcgov/entity#4812

*Description of changes:*
- updated Storybook to latest version
- updated folder structure per latest version
- created shared component "DetailComment"

*Notes:*

Created npm organization [https://www.npmjs.com/settings/bcrs-shared-components/packages](https://www.npmjs.com/settings/bcrs-shared-components/packages), all packages will be published here.

- Login to npm `npm login`
- To publish new version execute `lerna version` -> change version, this will update version in package.json and commit those changes.
- Run `lerna publish from-package` to explicitly publish packages where the latest version is not present in the registry. Yet to figure out `lerna publish from-git`, good for automation.

To consume the npm package execute `npm i @bcrs-shared-components/detail-comment` in other app's

Note: This app can still be refined before using it.

Examples of Storybook are contained in "src/stories" folder.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).
